src/lib/utils/references.ts

```diff
@@ -43,7 +43,8 @@ export enum Platform {
     ServerJava = 'server-java',
     ServerDotNet = 'server-dotnet',
     ServerGraphql = 'server-graphql',
-    ServerRest = 'server-rest'
+    ServerRest = 'server-rest',
+    ServerGo = 'server-go'
```

Comment:
Let's not put Go last. GraphQL and REST should be last because they're not really SDKs. I messaged Eldad to see if he had any preference on the order.